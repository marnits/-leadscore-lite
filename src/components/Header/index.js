import React from 'react';
import PropTypes from 'prop-types';
import styles from './styles.module.scss';

const Header = ({ text }) => (
  <span className={styles.Header}>{text}</span>
);

Header.propTypes = {
  text: PropTypes.string.isRequired,
};

export default Header;
