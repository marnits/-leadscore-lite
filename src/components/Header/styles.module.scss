@import "../../utils/styles/colors.scss";

.header {
  composes: cut-text from "../../utils/styles/cut-text.scss";
  color: $ink;
  font-weight: 700;
  font-size: 24px;
  line-height: 32px;
  margin: 0;
  width: 100%;
}
