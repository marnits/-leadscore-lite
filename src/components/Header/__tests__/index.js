import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../index';

describe('Header', () => {
  test('should render component properly', () => {
    const tree = renderer.create(<Header text="header" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
