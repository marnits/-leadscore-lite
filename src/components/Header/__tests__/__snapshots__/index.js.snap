// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render component properly 1`] = `
<span
  className="Header"
>
  header
</span>
`;
